data/resources/style.css

```diff
@@ -42,7 +42,7 @@
   font-weight: bold;
   min-width: 0.7em;
   border-radius: 10px;
-  padding: 2px 5px;
+  padding: 2px 5px 1px;
```

Comment:
Here, at least, this is still not centered.
I have to set this to 0px instead

0px (left) vs 1px (right)
![Bildschirmfoto von 2022-01-06 23-08-59](https://user-images.githubusercontent.com/3630213/148459523-f4ffeca6-36d6-4e44-a54a-e096a428f8f4.png)

Reply:
Right! Thanks for noticing that.